docs/sql-guide/data-types/data-type-timestamp.md

```diff
@@ -35,6 +36,39 @@ TIMESTAMP [TIMEUNIT {value}]
 
 ## Examples
 
-### CREATE TABLE with all data types
+### Implicitly convert integer to timestamp
```

Comment:
content moved here from TOTIMESTAMP() because it is explicitly about timestamp and doesn't feature anything with TOTIMESTAMP()